ui.py

```diff
@@ -1319,8 +1328,7 @@ def draw_item(self, layout, context, item):
                                          "lpe:(C<T[S]%G>[DS]+<L.%LG>)|(C<T[S]%G>[DS]*O)"]:
                 col.prop(item, "denoise_aov")
```

Comment:
When would you grey it out though?  I suppose the warning can just be removed.....